cluster/gce/gci/configure.sh

```diff
@@ -728,27 +732,6 @@ function install-kube-binary-config {
   rm -f "${KUBE_HOME}/${server_binary_tar}.sha1"
 }
 
-function ensure-mizar-kernel-and-ifname() {
```

Comment:
We should keep this option for now. It is non-default but if we run into any blocking problems in ubuntu 20.04 we can fall back to this for debugging or verification.